openapi/4.0/README.adoc

```diff
@@ -0,0 +1,41 @@
+---
+title: "MicroProfile Open API 4.0 (under development)"
+date: To Be Determined
+summary: "Release for OpenAPI 3.1 Spec Support"
+---
+MicroProfile Open API aims to provide a set of Java interfaces and programming models which allow Java developers to natively produce OpenAPI v3.0 and v3.1 documents from their Jakarta REST applications.
+
+The goal of this release is to provide features requested by the community, with a focus on adding support for the OpenAPI v3.1 specification. Changes targeting the release include:
+
+=== API/SPI Changes
+
+* Changes for OpenAPI Specification (OAS) v3.1 (https://github.com/eclipse/microprofile-open-api/issues/333[333])
+** https://github.com/eclipse/microprofile-open-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22OAS+3.1.0%22+milestone%3A%22MP+OpenAPI+4.0%22[Detailed listing of issues for OAS v3.1 in the MP OpenAPI 4.0 milestone]
+* Update JAX-RS to Jakarta RESTful Web Services with Jakarta EE 10 alignment (https://github.com/eclipse/microprofile-open-api/issues/574[574])
+
+=== Other Changes
+
+* Make request body required by default (https://github.com/eclipse/microprofile-open-api/issues/349[349])
+* MP Parent version deviates from MP Parent TCK BOM version (https://github.com/eclipse/microprofile-open-api/issues/573[573])
+* A configuration example in microprofile-openapi-spec.asciidoc is missing a comma which causes schema parse error (https://github.com/eclipse/microprofile-open-api/issues/568[568])
+
+
+The release will not contain any backwards incompatible changes - to be determined.
```

Comment:
We're pretty sure now that this release will contain backwards incompatible changes - they're impossible to avoid while updating the model to match OAS 3.1.